site/app/templates/course/EditCourseMaterialsForm.twig

```diff
@@ -0,0 +1,116 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-course-materials-form{% endblock %}
+{% block title %}Edit Course Materials{% endblock %}
+{% block body %}
+    <div id="material-edit-form" class="flex-col flex-col-space" data-directory = "Check">
+        <label id="edit_sections">
+            <p class="black-message">
+                Do you want to restrict this course material to some sections?
+            </p>
+            <label class = "radio">
+                <input id = "all_Sections_Showing_No" name="show_Some_Section_Selection_Edit" type = "radio" onchange="toggleRegistrationSectionsEdit()" checked>
+                No
+            </label>
+            <label class = "radio">
+                <input id = "all_Sections_Showing_Yes" name="show_Some_Section_Selection_Edit" type = "radio" onchange="toggleRegistrationSectionsEdit()">
+                Yes
+            </label>
+            <div id = "show_Some_Section_Selection_Edit" style="display: none">
+            {% for section in reg_sections %}
+                <tr>
+                    <span class = "inline">
+                        <td>
+                            Section {{ section['sections_registration_id'] }}
+                            <input id="section-edit-{{section['sections_registration_id']}}" type="checkbox" value="off"/>
+                        </td>
+                        <td>
+                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                        </td>
+                    </span>
+                </tr>
+            {% endfor %}
+            </div>
+            <script type="text/javascript">
+                function toggleRegistrationSectionsEdit(){
+                    $("#show_Some_Section_Selection_Edit").toggle();
+                } 
+            </script>
+        </label>
+        <label id="upload_dt">
+            <p>
+                <strong>Choose a time to release the files being uploaded:</strong>
+            </p>
+            <input id="edit_picker" class="edit_picker" type="text" size="26"/>
+        </label>
+        <label>
+            <input id="hide-materials-checkbox-edit" type="checkbox"/>
+            Hide from students (Note: If checked, students will not be able to see course matierals, but can still access them via the URL)
+        </label>
+        <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+    </div>
+    <script type="text/javascript">
+        function submitEdit(){ // Submit button
+                var sectionsEdit = [];
+                var cmTime = document.getElementById("edit_picker").value;
+                var requestedPath = document.getElementById("material-edit-form").getAttribute("data-directory");
+                var hideFromStudents = 'off';
```

Comment:
You `let` for defining variables in JS. We are moving away from `var` as possible.